README.md

```diff
@@ -91,6 +93,27 @@ Now `zeek -NN _Zeek::Spicy` should show similar output as above.
 When you run Zeek, add `spicy-analyzers` to the command line to load
 the analyzer scripts.
 
+## Configuration
+
+By default, all included analyzers will be activated, they will
+automatically disable any standard analyzers that they replace. If you
+want to disable one of the Spicy analyzers, you can do so by calling
+one of the built-in functions
+[disable_protocol_analyzer/disable_file_analyzer()](https://docs.zeek.org/projects/spicy/en/latest/zeek.html#functions).
+For example, to disable the HTTP analyzer, add this to your
+`site.zeek`:
+
+    event zeek_init()
+        {
+        Spicy::disable_protocol_analyzer(Analyzer::ANALYZER_SPICY_HTTP);
+        }
+
+You can find the `ANALYZER_*` value to use for an analyzer in the
+output of `zeek -N _Zeek::Spicy`.
+
+(Note that `disable_file_analyzer()` requires a current development
```

Comment:
Can we reference a SHA here?

Reply:
Not worth it I think, people following Zeek master should keep updating anyways, and not stick to an old version.